### ISSUE TITLE
Disable [Enabled] "Desktop Settings" Profile Causes Logitech "Device Manager" Not To Connect

### ISSUE THREAD

**Issue by incryptx** [antonpup/Aurora#2110]
Version 0.8.1

The profiles on the left side of Aurora, which have dialog sub-menu choices of "Enabled", "Overlays Enabled", "Hidden", if you uncheck the "Enabled" box for the Desktop Profile, this will kill the Logitech Device Manager from "Connecting" and it will remain "Not Initialized" 

To demonstrate this, Open Aurora, go to Settings" and then "Device Manager". The first item in this menu is for Logitech. Assuming the Logitech Device is "Connected", right click on the Desktop Settings Icon on the left, and uncheck "Enabled". You will see the Logitech Device go from "Connected" to "Not Initialized". 

My conclusion is that if the Desktop Settings Icon sub-menu "Enabled" item is unchecked, the Logitech Device Manager will never connect, no matter what you do. It also seems that Aurora on the whole, is dependent on the "Desktop Settings" Icon being available, and having the Desktop Profile "Enabled" at all times.

For my own personal preference, I went through all of the game profiles on the left of Aurora's screen and unchecked all the sub-menu items for all the game profiles I do not use or need, as well as hiding all of them from view. It was only until I tried to do this with the default "Desktop" profile, did I realize that unchecking the "Enabled" box, that this would cause the Logitech Device to move from "Connected" to "Not Initialized" You can get the settings menu open and check and uncheck the "Enabled" item on the Desktop Profile, and watch it change states. 

So, whatever you do, DO NOT UNCHECK "ENABLED" on the Desktop Profile or you will be unable to connect the Logitech Device (and I assume any device) in Device Manager.

**Comment by diogotr7** [comment]
This is intended behaviour. The objective is allowing the user to get aurora's ingame integrations when in game, but keep the stock lights when not in game. some users prefer razer synapse's effects to aurora ones for example. 

If you have desktop disabled but a game enabled, aurora should connect to the devices once it detects you're in game, and disconnect when you leave the game again.

**Comment by incryptx** [comment]
Ok. But if you have the desktop profile disabled, and depend on it connecting when you connect to the game and then disconnect when you leave the game, no one would ever see this happen and the device would always show "Not Initialized", leading folks to believe it was not working, as it wold connect and disconnect while entering or exiting the game. I spent hours trying to find out why the logitech device never connected, only to discover I had inadvertantly disabled it by unchecking the "enabled" box on the desktop profile, which I did not realize was preventing me from seeing any connection status.

One of those things that should be documented, No one would ever realize a device in device manager was connecting and disconnecting on game entry or exit behind the scenes as ithe device would always show "Not Intitialized". 

(On a different topic, and why I am playing with Aurora settings and devices.)

So, here I am trying to get the Elite Game Profile to run, as it will load to the keyboard, but the blue keyboard animation it used to do when I "Jumped" or went to supercruise no longer animates. I have tried everything I can do to fix it. It must be the Elite Profile itself that is malfunctioning. I have taken the profile.json from each version of Aurora and tried them all, but no animations occur with this profile.
